blog-article.html

```diff
@@ -4,12 +4,329 @@
+            <p class="article-text">Alternatively, if you’re on nft.binance.com, go to User Center > Settings > Basic >
+                Set Avatar and Nickname, and you’ll be redirected to the main site. Binance NFT Creators have the option
+                of setting their Avatars on nft.binance.com. Once you set your new unique nickname, it will be
+                standardized and used across the Binance ecosystem.</p>
+        </article>
+        <article class="blog-details">
+            <h2 class="article-heading">NFT Avatar Generators</h2>
+            <p class="article-text">You can find a range of NFT marketplaces where you can buy existing NFTs or list
+                your own NFTs for sale.</p>
+        </article>
+        <section class="blog-caption">
+            <h3 class="hidden-heading">TAGS</h3>
+            <div class="caption=tags">
+                <p class="caption-name">TAGS:</p>
+                <ul class="tag-list">
+                    <li class="tag-item">ICO</li>
+                    <li class="tag-item">CYPPUNK</li>
+                    <li class="tag-item">LIGHT</li>
+                </ul>
+            </div>
+            <div class="caption-share">
+                <p class="caption-name">SHARE:</p>
+                <ul class="social-list">
+                    <li class="social-icon">
+                        <a href="#" class="social-link">
+                            <img src="#" alt="Discord">
+                        </a>
+                    </li>
+                    <li class="social-icon">
+                        <a href="#" class="social-link">
+                            <img src="#" alt="Twitter">
+                        </a>
+                    </li>
+                    <li class="social-icon">
+                        <a href="#" class="social-link">
+                            <img src="#" alt="YouTube">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <div class="article-browsing">
+            <a href="#" class="previous-article">
+                <figure class="article-figure">
+                    <img src="#" alt="playing card">
+                    <figcaption class="article-caption">
+                        <p class="caption-text">PREVIOUS</p>
+                        <p class="sub-text">Senior Apple employee alleges sexism at</p>
+                    </figcaption>
+                </figure>
+            </a>
+            <a href="#" class="next-article">
+                <figure class="article-figure">
+                    <img src="#" alt="cat">
+                    <figcaption class="article-caption">
+                        <p class="caption-text">NEXT</p>
+                        <p class="sub-text">Senior Apple employee alleges sexism at</p>
+                    </figcaption>
+                </figure>
+            </a>
+        </div>
+        <section class="sidebar">

```

Comment:
Use the `<aside>` tag for content that is tangentially related to the main content. [Learn more about the `<aside>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).